nixos/modules/services/networking/wakeonlan.nix

```diff
@@ -31,6 +31,21 @@ in
 
     services.wakeonlan.interfaces = mkOption {
       default = [ ];
+      type = types.listOf (types.submodule { options = {
+        interface = mkOption {
+          type = types.str;
+          description = "Interface to enable for Wake-On-Lan.";
+        };
+        method = mkOption {
+          type = types.enum [ "magicpacket" "password"];
+          description = "Interface to enable for Wake-On-Lan.";
+        };
+        password = mkOption {
+          default = "";
```

Comment:
```suggestion
```

Reply:
why remove the default?

Reply:
Having the default would cause this to build but not work if you specify method = "password", but no password